### DISCUSSION TITLE
Launching webaps from command line

### DISCUSSION THREAD

**Discussion by lukas-remis** [filips123/PWAsForFirefox#240] [Q&A] [answered]
Hi,
How can I install an webapp in Ubuntu for all users? And how to launch an app from command line?

Thanks,
Lukas

**Comment by filips123** [comment, marked as answer]
Unfortunately, it's not possible to automatically install a web app for all users, due to complicated permission handling and some other stuff. However, there is a way to copy all web apps from one user to another:

1. Copy `config.json` from one user to another (located in `$XDG_DATA_HOME/firefoxpwa/` or `$HOME/.local/share/firefoxpwa/`).
2. Also copy the `runtime` directory to another user.
3. If you also want to copy web app data, also copy the `profiles` directory from the same base directory.
4. Log in to another user and run "Update web apps" from the extension settings. This will create web app shortcuts and icons. Alternatively, you could instead manually copy all icons and shortcuts from one user to another.


To launch an app from the command line, you can use `firefoxpwa site launch ID`. To get the ID, you can run `firefoxpwa profile list` and the web app ID will be in parentheses.
